contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -549,7 +549,7 @@ namespace eosiosystem {
           * @param header - the block header produced.
           */
          [[eosio::action]]
-         void onblock( ignore<block_header> header );
+         void onblock( const ignore<block_header>& header );
```

Comment:
Actually, it's probably better to keep the `ignore` types by value since they are empty structs.

Reply:
Done.